# Message to control high level actions of the Ego-Vehicle


# desired_speed : Desired speed of the vehicle

float32 desired_speed


